src/DesktopVirtualization/help/Remove-AzWvdUserSession.md

```diff
@@ -55,7 +55,7 @@ Accept wildcard characters: False
 ```
 
 ### -Force
-Force flag to login off userSession.
+Force flag to force userSession deletion
```

Comment:
```suggestion
Specify to force userSession deletion
```
Please update this parameter description in directive as well, otherwise, your change will be overridden in next iteration. Please follow this [guide](https://github.com/Azure/autorest.powershell/blob/main/docs/directives.md#parameter-description)